packages/rancher-prom-to-teams/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,53 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -2,9 +2,19 @@
+ # This is a YAML-formatted file.
+ # Declare variables to be passed into your templates.
+ 
++global:
++  cattle:
++    systemDefaultRegistry: ""
++
++nameOverride: "prom2teams"
++fullnameOverride: ""
++
++namespace: rancher-prom2teams
++replicaCount: 1
++
+ image:
+   repository: idealista/prom2teams
+-  tag:
++  tag: 3.2.1
+   pullPolicy: IfNotPresent
+ 
+ resources:
+@@ -24,7 +34,10 @@
+   port: 8089
+   connector:
+   connectors: {}
++  # group_alerts_by can be one of
++  # ("name" | "description" | "instance" | "severity" | "status" | "summary" | "fingerprint" | "runbook_url")
+   group_alerts_by:
++  # loglevel can be one of (DEBUG | INFO | WARNING | ERROR | CRITICAL)
+   loglevel: INFO
+   templatepath: /opt/prom2teams/helmconfig/teams.j2
+   config: /opt/prom2teams/helmconfig/config.ini
+@@ -41,3 +54,18 @@
+   fsGroup: 65534
+   # readOnlyRootFilesystem is a flag to enable readOnlyRootFilesystem for the Hazelcast security context
+   readOnlyRootFilesystem: true
++
++nodeSelector: { }
++
++tolerations: [ ]
++
++affinity: { }
++
++preInstallHook:
++  securityContext:
++    runAsNonRoot: true
++    runAsUser: 1000
++  image:
++    repository: rancher/rancher-agent
++    tag: v2.5.7 # TODO: change to v2.5.8
```

Comment:
I don't think we need to hard code this at all, right?

Reply:
You are right, I can make this configurable in the parent chart's value.yaml 

The need here is that we need an image where we can run the shell script, `rancher/rancher-agent` is what the monitoring chart uses.